web-ui/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I guess this isn't supposed to be committed, you could ignore the `.vscode/` directory.